tests/integration/kubernetes/gha-run.sh

```diff
@@ -173,21 +173,6 @@ function delete_cluster() {
 }
 
 function get_nodes_and_pods_info() {
-    describe_pods="${1:-"no"}"
-
-    echo "::group::Get node information"
-    kubectl get nodes -o wide --show-labels=true
-    echo "::endgroup::"
-    echo ""
-    echo "::group::Get all the pods running"
-    kubectl get pods -A
-    echo "::endgroup::"
-    echo ""
-    if [[ "${describe_pods}" == "yes" ]]; then
-	echo "::group::Describe all the pods"
-    	kubectl describe pods -A
```

Comment:
I believe the describe pods commands could give useful logs when there is a pod failure. You may want to reconsider this. In case of failure, the describe command ends up being quite helpful.

Reply:
We're already getting the `kubectl describe pods`  as part of a failure, this one was getting that from **all** pods in the cluster after the tests finished, and I used that to debug a few issues, but that has proven not so useful in that scenario.

The `describe pod` in case of failures is kept, and I 100% agree with you that those ones should be kept there.

Does that sound reasonable?